roles/bootstrap-os/tasks/main.yml

```diff
@@ -41,13 +43,13 @@
 
 - name: Assign inventory name to unconfigured hostnames (non-CoreOS and Tumbleweed)
```

Comment:
This title is actually a lie - it is not limited to OpenSUSE Tumbleweed but all OpenSUSE systems and also will exclude ClearLinux.